app/controllers/admin/videos_controller.rb

```diff
@@ -0,0 +1,24 @@
+class Admin::VideosController < AdminsController
+  def new
+    @video = Video.new
+  end
+
+  def create
+    @video = Video.new(video_params)
+    category = Category.find(params[:video][:categories]) if params[:video][:categories]
+    @video.categories << category
```

Comment:
At the very first beginning I though the category and the video should be many-to-many relationship. So the codes follows the many-to-many settings. In the add new video since the select field can only input single selection, so I work around the code to just connect one category with the newly added video.

The many-to-many category and video scheme does not bother me till the add new video section. Should I rewrite the models to one-to-many setups as the solution?